config/initializers/sentry.rb

```diff
@@ -0,0 +1,12 @@
+Sentry.init do |config|
+  config.dsn = 'https://e43549eae9bb4f81891972b87e79b5ee@o1209445.ingest.sentry.io/6356423'
```

Comment:
It's probably best to keep this out of the repo. It's not a big deal but in theory, people could use this to connect it to their own project and send us fake errors...
Can you make this come from an environment variable instead?

Reply:
Yes, will change to env